src/components/TodoItem.js

```diff
@@ -1,10 +1,14 @@
 import React from 'react';
 import PropTypes from 'prop-types';
+import CN from 'classnames';
```

Comment:
use lowercase letters
```suggestion
import cn from 'classnames';
```